translator/isValid.go

```diff
@@ -63,3 +63,12 @@ func ResetMessages() {
 	ErrorMessages = make([]string, 0)
 	InfoMessages = make([]string, 0)
 }
+
+// ValidDays represents the valid possible values for retentionInDays
+var ValidDays = map[int]bool{1: true, 3: true, 5: true, 7: true, 14: true,
```

Comment:
Should -1 included in the list or not? Also if every entry in this map has a value true, why not using a Set instead?

Reply:
It seems like Golang doesn't have builtin Sets, but some of our modules have defined Sets (e.g. https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#String) that we can use.